src/Analyzer.JsonRuleEngine.UnitTests/InequalityOperatorTests.cs

```diff
@@ -101,63 +101,39 @@ public void GetName_ReturnsCorrectName()
+        [DataRow("2021-02-28", "2021-09-20", true, false, false, DisplayName = "A date is not greater than another date")]
+        [DataRow("2021-02-28", "2021-02-28", true, false, false, DisplayName = "A date is not greater than another date because both are equal")]
         // <date
-        [DataRow(637500672000000000, 637676928000000000, false, false, true, DisplayName = "A date is less than another date")]
-        [DataRow(637676928000000000, 637500672000000000, false, false, false, DisplayName = "A date is not less than another date")]
-        [DataRow(637676928000000000, 637676928000000000, false, false, false, DisplayName = "A date is not less than another date because both are equal")]
+        [DataRow("2021-02-28", "2021-09-20", false, false, true, DisplayName = "A date is less than another date")]
+        [DataRow("2021-09-20", "2021-02-28", false, false, false, DisplayName = "A date is not less than another date")]
+        [DataRow("2021-09-20", "2021-09-20", false, false, false, DisplayName = "A date is not less than another date because both are equal")]
         // >=date
-        [DataRow(637676928000000000, 637500672000000000, true, true, true, DisplayName = "A date is greater or equal to another date")]
-        [DataRow(637500672000000000, 637676928000000000, true, true, false, DisplayName = "A date is not greater or equal to another date")]
-        [DataRow(637676928000000000, 637676928000000000, true, true, true, DisplayName = "A date is greater or equal to another date because both are equal")]
+        [DataRow("2021-09-20", "2021-02-28", true, true, true, DisplayName = "A date is greater or equal to another date")]
+        [DataRow("2021-02-28", "2021-09-20", true, true, false, DisplayName = "A date is not greater or equal to another date")]
+        [DataRow("2021-09-20", "2021-09-20", true, true, true, DisplayName = "A date is greater or equal to another date because both are equal")]
         // <=date
-        [DataRow(637500672000000000, 637676928000000000, false, true, true, DisplayName = "A date is less or equal to another date")]
-        [DataRow(637676928000000000, 637500672000000000, false, true, false, DisplayName = "A date is not less or equal to another date")]
-        [DataRow(637500672000000000, 637500672000000000, false, true, true, DisplayName = "A date is less or equal to another date because both are equal")]
-        public void EvaluateExpression_ValidDateType_ReturnsExpectedEvaluationResult(long leftTicks, long rightTicks, bool greater, bool orEquals, bool evaluationResult)
+        [DataRow("2021-02-28", "2021-09-20", false, true, true, DisplayName = "A date is less or equal to another date")]
+        [DataRow("2021-09-20", "2021-02-28", false, true, false, DisplayName = "A date is not less or equal to another date")]
+        [DataRow("2021-02-28", "2021-02-28", false, true, true, DisplayName = "A date is less or equal to another date because both are equal")]
```

Comment:
Add tests for the other supported formats. At least one for each.

Reply:
I added test cases like that in RuleParsingTests, do you think those should be "moved" here or it makes sense to have both?

Reply:
I think there should still be a test here to show that that format actually gets evaluated correctly.

Reply:
Thank you! It made me catch an error